src/mcp/utils/auth.ts

```diff
@@ -44,25 +44,69 @@ export class DevCycleAuth {
             this._authToken = await this.apiAuth.getToken(flags, this._orgId)
 
             if (!this._authToken) {
-                throw new Error(
-                    'No authentication found. Please set DEVCYCLE_CLIENT_ID and DEVCYCLE_CLIENT_SECRET environment variables, ' +
-                        'or run "dvc login sso" in the CLI first.',
-                )
+                const hasEnvVars =
+                    process.env.DEVCYCLE_CLIENT_ID &&
+                    process.env.DEVCYCLE_CLIENT_SECRET
+
+                if (hasEnvVars) {
+                    throw new Error(
+                        'Authentication failed with provided environment variables. ' +
+                            'Please verify your DEVCYCLE_CLIENT_ID and DEVCYCLE_CLIENT_SECRET are correct, ' +
+                            'or run "dvc login sso" to authenticate with SSO.',
+                    )
+                } else {
```

Comment:
you don't need an else here - you're throwing

Reply:
yea, reads cleaner this way thou.